sensors:
  depth_camera_0:
    type: depth_camera
    pose:
      xyz: [3.082385, 1.750927, 1.82]
      rpy: [0, 'pi/2', 0]

  logical_camera_0:
    type: logical_camera
    pose:
      xyz: [5, 1.750927, 1.82]
      rpy: [0, 'pi/2', 'pi/2']

  logical_camera_1:
    type: logical_camera
    pose:
      xyz: [3.082385, 3.641234, 2.49]
      rpy: [0, 'pi/2', 'pi/2']

  depth_camera_1:
    type: depth_camera
    pose:
      xyz: [5, 3.641234, 2.49]
      rpy: [0, 'pi/2', 0]

  laser_profiler_0:
    type: laser_profiler
    pose:
      xyz: [0., 1.486, 1.526]
      rpy: ['pi/2', 'pi/2', 0]

  range_finder_0:
    type: proximity_sensor
    pose:
      xyz: [0, 2.84, 0.97]
      rpy: ['pi/2', 0.6, 'pi/2']

  breakbeam_0:
    type: break_beam
    pose:
      xyz: [-0.26, 3.36, 0.94]
      rpy: [0, 0, 0]
